### DISCUSSION TITLE
游戏大模型-第二人生

### DISCUSSION THREAD

**Discussion by woxinyonghen** [InternLM/Tutorial#480] [project]
基于Internlm2的人生重开模拟器大模型版

![bandicam 2024-01-23 00-16-03-599 00-03-01 20240123-110759645](https://github.com/InternLM/tutorial/assets/23326162/938522b1-595b-475d-8faf-b73aaa4effda)

**Comment by leonfrank** [comment]
请问队伍组满了嘛，我可以加入吗？

**Comment by leonfrank** [comment]
我微信号是fyl19941230
